sdk/identity/identity/src/msal/nodeFlows/nodeCommon.ts

```diff
@@ -170,7 +207,11 @@ To work with multiple accounts for the same Client ID and Tenant ID, please prov
    * Clears MSAL's cache.
    */
   async logout(): Promise<void> {
-    // Intentionally empty
+    const cache = await this.publicApp?.getTokenCache();
```

Comment:
do we not need to do this for the confidentialApp as well?

Reply:
Nothing is currently using logout() and it is not part of the public API. We should really remove this code.

Reply:
Removed.